homeassistant/components/lutron/__init__.py

```diff
@@ -128,6 +131,11 @@ def should_poll(self):
         """No polling needed."""
         return False
 
+    @property
+    def unique_id(self):
+        """Return the unique ID of the device."""
+        return f"{DOMAIN}_{self._controller.guid}_{ getattr(self._lutron_device, 'uuid', '')}_{getattr(self._lutron_device, 'id', '')}"
```

Comment:
Why include the integration ID as well?

Reply:
It's a fair point. I don't actually understand how Lutron uses the UUID sufficiently. Unfortunately, not everything in RadioRA2 has a serial number that's exposed, and the official guidance is to use the integration ID. I'll remove it locally and see how it works with my setup. I have a variety of devices, but probably not everything.